generated/nifpga/nifpga_library.cpp

```diff
@@ -139,6 +141,14 @@ NiFpga_Status NiFpgaLibrary::Abort(NiFpga_Session session)
   return function_pointers_.Abort(session);
 }
 
+NiFpga_Status NiFpgaLibrary::AcknowledgeIrqs(NiFpga_Session session, uint32_t irqs)
+{
+  if (!function_pointers_.AcknowledgeIrqs) {
+    throw nidevice_grpc::LibraryLoadException("Could not find NiFpga_AcknowledgeIrqs.");
```

Comment:
I just noticed this, and this should not be fixed in this PR but should this read `Could not find NiFpgaDll_AcknowledgeIrqs`? You might need to fix something in codegen to fix this error.

Reply:
This is not a blocking comment for this review. Feel free to resolve this and merge the change, but do check if this suggestion makes sense.

Reply:
Sure, I'll take care of this in follow-up